bricks-maven-plugin/src/main/java/com/github/ingogriebsch/bricks/maven/plugin/AnalyzeMojo.java

```diff
@@ -0,0 +1,98 @@
+/*-
+ * #%L
+ * Bricks Maven Plugin
+ * %%
+ * Copyright (C) 2018 - 2019 Ingo Griebsch
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.github.ingogriebsch.bricks.maven.plugin;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.AnalyzerContext;
+import com.github.ingogriebsch.bricks.maven.plugin.analyzer.discovery.Analyzers;
+import com.github.ingogriebsch.bricks.model.Component;
+
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+import lombok.SneakyThrows;
+
+/**
+ * Goal that analyzes a Project using a bunch of discovered Analyzers
+ * 
+ */
+@Mojo(name = "analyze", threadSafe = false)
```

Comment:
not bound at all by default. it is supposed to be possible to configure it, without necessarily triggering it within the maven lifecycle, so that you can run it manually, if you like.

Reply:
plz. resolve if you agree

Reply:
just in case: note that you can run it with -Dstdout=true (parameter name is supposed to change, of course) to see what the bricks.json would look like.

Reply:
Even if you bind it to a phase you can execute it standalone (but would help to avoid to configure it explicitely every time). But for me okay for now. Let us wait and see which phase will get the 'right one' after we implemented some more analyzers.

Reply:
you can always run it standalone. my concern is to not be able to configure it (dependencies for instance) and then being not able to *skip* it. Of cource, we could add skipAnalysis, but why?